modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -14,6 +14,7 @@ Before upgrading a cluster with manually maintained credentials, you must create
 . Extract and examine the `CredentialsRequest` custom resource for the new release.
 +
 The "Manually creating IAM" section of the installation content for your cloud provider explains how to obtain and use the credentials required for your cloud.
+//This probably needs to be fleshed out better and/or have the reference updated due to ccoctl content reorganization.
```

Comment:
This section should point at the `cco-ccoctl-upgrading.adoc` flow and this section depend on CredentialsRequest` extraction.  Possibly worth a re-org so that's its own section which both `ccoctl` consumers and folks who want to manually do what `ccoctl` does can conveniently consume it?

Reply:
I think there is a wording problem here that is making this less clear. This procedure is for manual _without_ the `ccoctl` to help. I think I will pull in the cloud-specific commands instead of sending them all over the place, like in step 5 for `ccoctl` clusters [here](https://62148--docspreview.netlify.app/openshift-enterprise/latest/updating/preparing_for_updates/preparing-manual-creds-update.html#cco-ccoctl-upgrading_preparing-manual-creds-update). Will also think about the intro paragraph to see if I can tease this out better and make sure users know which steps they care about for their cluster.